.github/workflows/quri-sdk-package.yml

```
@@ -117,7 +117,7 @@ jobs:
             target: "aarch64-apple-darwin"
           - os: "windows-latest"
             target: "x86_64-pc-windows-msvc"
-        python-version: [ "11" ]
+        python-version: [ "9", "11", "13" ]
```

Why do you leave out "10" and "12"?

---

It is for reducing CI runs. We can adding 10 and 12 if it is feasible.
```suggestion
        python-version: [ "9", "10", "11", "12",  "13" ]
```

---

Ah, I see. Testing both the sdist package takes between 3 and 11 minutes for each run depending on the runner. So, I think adding another python version adds a total of ~ 25 min of runtime.... I don't think we have a concrete guideline on how much runtime to spend, so I think it is up to you.

---

Thx. IMHO it is ok to go without the fix.